e2e_test/sink/cassandra_sink.slt

```diff
@@ -0,0 +1,33 @@
+statement ok
+CREATE TABLE t6 (v1 int primary key, v2 smallint, v3 bigint, v4 real, v5 float, v6 varchar, v7 date, v8 timestamp, v9 boolean);
```

Comment:
Use timestamptz for cassandra.
https://cassandra.apache.org/doc/stable/cassandra/cql/types.html#timestamps
> it is recommended that the time zone always be specified for timestamps when feasible.